bzt/modules/apiritif/generator.py

```diff
@@ -653,6 +661,50 @@ def _gen_action(self, action_config):
 
         return [ast.Expr(element) for element in action_elements]
 
+    def _gen_answer_dialog_mngr(self, type, value):
+        if not type in ['prompt', 'confirm']:
```

Comment:
here and in another places `if type not in..`  would be better.

Reply:
Thanks, I will fix it